### PR TITLE
bugfix/payout_bankwire_bug fixed

### PR DESCRIPTION
**Pull request by SoloJr**

### CODE REVIEW

File: src/test/java/com/mangopay/core/BaseTest.java [hunk 1]
```diff
@@ -539,6 +539,7 @@ protected PayOut getJohnsPayOutBankWire() throws Exception {
             payOut.setDebitedWalletId(wallet.getId());
             payOut.setMeanOfPaymentDetails(new PayOutPaymentDetailsBankWire());
             ((PayOutPaymentDetailsBankWire) payOut.getMeanOfPaymentDetails()).setBankAccountId(account.getId());
+            ((PayOutPaymentDetailsBankWire) payOut.getMeanOfPaymentDetails()).setPayoutModeRequested(PayoutMode.STANDARD);
```
Comment (fredericdelordm) [inline src/test/java/com/mangopay/core/BaseTest.java:542]: @SoloJr Does it take into account "INSTANT_PAYMENT" value for the PayoutModeRequested parameter ?

File: src/main/java/com/mangopay/core/deserializer/PayOutDeserializer.java [hunk 2]
```diff
@@ -19,8 +19,10 @@ public PayOut deserialize(JsonElement json, Type typeOfT, JsonDeserializationCon
                     meanOfPaymentDetails.setBankAccountId(object.get("BankAccountId").getAsString());
                 if (object.has("BankWireRef") && !object.get("BankWireRef").isJsonNull())
                     meanOfPaymentDetails.setBankWireRef(object.get("BankWireRef").getAsString());
-                if (object.has("PayoutModeRequested") && !object.get("PayoutModeRequested").isJsonNull())
-                    meanOfPaymentDetails.setPayoutModeRequested(PayoutMode.valueOf(object.get("PayoutModeRequested").getAsString()));
+                if (object.has("ModeRequested") && !object.get("ModeRequested").isJsonNull())
+                    meanOfPaymentDetails.setPayoutModeRequested(PayoutMode.valueOf(object.get("ModeRequested").getAsString()));
+                if (object.has("ModeApplied") && !object.get("ModeApplied").isJsonNull())
```
Comment (fredericdelordm) [inline src/main/java/com/mangopay/core/deserializer/PayOutDeserializer.java:24]: @SoloJr https://docs.mangopay.com/guide/instant-payment-payout

Do we take into account Status && FallbackReason ?

**Review by fredericdelordm — CHANGES_REQUESTED** [review]
Hello @SoloJr I have added some questions

File: src/test/java/com/mangopay/core/BaseTest.java (hunk 1 above)
Comment (SoloJr) [reply src/test/java/com/mangopay/core/BaseTest.java:542]: @fredericdelordm yes it does, but I cannot test this case due to API limitations ("Payout author is not Kyc validated for InstantPayment")

**Review by fredericdelordm — APPROVED** [review]

**Pull request merged**